src/edu/pugetsound/nlp/geoanalyzer/GeoAnalyzer.java

```diff
@@ -0,0 +1,24 @@
+package edu.pugetsound.nlp.geoanalyzer;
+
+import edu.pugetsound.mathcs.nlp.architecture_nlp.features.TextAnalyzer;
+import edu.pugetsound.mathcs.nlp.kb.KBController;
+import edu.pugetsound.mathcs.nlp.lang.Conversation;
+import edu.pugetsound.mathcs.nlp.lang.Utterance;
+public class GeoAnalyzer {
+	
+	private TextAnalyzer analyzer;
+	private KBController knowledge;
```

Comment:
Remove the KBController. There is no need for this program to access the knowledge base.